docs/Developers/Git_Basics.rst

```diff
@@ -62,8 +72,9 @@ First let's talk about remotes. To communicate with the outside world, git uses
 what are called remotes. These are repositories other than the one on your
 local disk which you can push your changes into (so that other people can see
 them) or pull from (so that you can get others changes).
-Now you should add a remote to your local machine so that you can ``pull`` and
-``push`` your commits. This can be simply done by using the command:
+Now you should add a remote of your fork to your local machine so that you can
+``pull`` and ``push`` your commits. This can be simply done by using the
+command:
 
 ::
 
```

Comment:
> See the line under here. It doesn't match what you have written. We want their fork to be "origin" so they should just clone from it. All the upstream stuff should actually go here.

1. The line below matches what I've written. And we have made a deliberate decision to move away from using `origin` to refer to remotes.

> If they add it as "myfork" they will not have a origin (default) remote, which will confuse git

2. This is not true. See: http://stackoverflow.com/a/9529518

> There's no requirement to name the remote repository origin: in fact the same repository could have a different alias for another developer.

We made a conscious decision against using `origin` because we wanted to be clear what each remote pointed to, and we didn't want to dictate how contributors should organize or name their remotes.